Dockerfile

```
@@ -34,41 +32,29 @@ RUN apt-get update \
     python2 python-is-python2 \
     && rm -rf /var/lib/apt/lists/*
 RUN ln -sf /usr/local/texlive/*/bin/* /usr/local/bin/texlive
+
 RUN tlmgr install texliveonfly \
-import \
-framed \
-type1cm \
-setspace \
-xcolor \
-enumitem \
-mathtools \
-ragged2e \
-float \
-algorithm2e \
-ifoddpage \
-relsize \
-algorithms \
-titlesec \
 microtype \
 etoolbox \
 changepage \
-trimspaces \
-datetime \
-fmtcount \
-xkeyval \
-bigfoot \
-mdframed \
-zref \
-needspace \
-ulem \
-footmisc \
-substitutefont \
-tempora \
+ragged2e \
+xcolor \
+mathtools \
+enumitem \
 tocloft \
+silence \
 background \
-everypage \
+xkeyval \
 pgf \
-babel-russian \
+everypage \
+csquotes \
+biblatex \
+biber \
 cyrillic \
-biblatex
-ENTRYPOINT ["/bin/bash", "-c"]
+babel-russian \
+lh
+
+COPY . /root/texmf/tex/latex/scn-latex-plugin
+RUN texhash
+
+ENTRYPOINT ["/bin/bash", "-c", "latexmk -pdf -bibtex tests.tex"]
```

```suggestion
ENTRYPOINT ["/bin/bash", "-c", "latexmk -pdf -bibtex"]
CMD ["tests.tex"]
```
I would love to see a more flexible approach to the entrypoint. Then it could be used to build any kind of file without major modifications simply by passing a different root file path to the command. 

We could also use `texliveonfly` as the default command, but because containers are ephemeral that would require setting up some caching for the latex plugins, and it may also produce some false positives (`texliveonfly ` could download a wrong package). 

It's a point of discussion: whether we adapt this image to suit most of the cases, or make it a good base to create another build image.

My point is that there are many works using scn-latex-plugins that are one-off projects (like small to medium scientific papers) that would benefit from having a generic-enough solution for building them. Otherwise users would have to create and define a build env every time, and while that might sound good for plugin management etc., I doubt most will go that route 😄